File: domain/src/main/java/no/nav/melosys/domain/dokument/arbeidsforhold/Utenlandsopphold.java
package no.nav.melosys.domain.dokument.arbeidsforhold;

import java.time.YearMonth;

import com.fasterxml.jackson.annotation.JsonProperty;
import no.nav.melosys.domain.dokument.felles.Periode;

public class Utenlandsopphold {

    private Periode periode;

    private String land;

    @JsonProperty("rapporteringsAarMaaned")
    private YearMonth rapporteringsperiode;

    /** Obs. Ikke kodeverk! */
    public String getLand() {
        return land;
    }

    public void setLand(String land) {
        this.land = land;
    }

    public Periode getPeriode() {
        return periode;
    }

    public void setPeriode(Periode periode) {
        this.periode = periode;
    }

    public YearMonth getRapporteringsperiode() {
        return rapporteringsperiode;
    }

    public void setRapporteringsperiode(YearMonth rapporteringsperiode) {
        this.rapporteringsperiode = rapporteringsperiode;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/dokument/arbeidsforhold/Utenlandsopphold.kt
package no.nav.melosys.domain.dokument.arbeidsforhold

import com.fasterxml.jackson.annotation.JsonProperty
import no.nav.melosys.domain.dokument.felles.Periode
import java.time.YearMonth


class Utenlandsopphold {
    var periode: Periode? = null

    /** Obs. Ikke kodeverk!  */
    var land: String? = null

    @JsonProperty("rapporteringsAarMaaned")
    var rapporteringsperiode: YearMonth? = null
}
